menubar.sh

```diff
@@ -1,15 +1,47 @@
 #!/bin/bash
 
+OS=$(uname -s)
 SERVICE=growler.py
-OUTFILE=~/Library/Logs/SecurityGrowler.log
+OUTFILE=/tmp/SecurityGrowler.log
```

Comment:
On most distros writing to `/var/log` would require root permission, or some other unprivileged-fork voodoo as far as I know.
Another viable alternative could be `~/.logs`, but that's not really a standard too.